src/modal/Modal.ts

```diff
@@ -254,6 +255,7 @@ export class Modal extends OmniElement {
                     background: transparent;
                     box-shadow: var(--omni-modal-container-box-shadow);
                     max-width: var(--omni-modal-max-width,100%);
+                    min-width: var(--omni-modal-min-width, auto);
```

Comment:
Is this backward compatible? Shouldn't it be unset instead of auto?

Reply:
According to MDN reference documentation the default for this is auto welcome to check the following link for reference. 

https://developer.mozilla.org/en-US/docs/Web/CSS/min-width

Reply:
I suspect `unset` would equate to `auto`. Happy either way.